executor/logger/worker.go

```diff
@@ -56,9 +76,54 @@ type Worker struct {
 	SdepName      string
 	Namespace     string
 	PredictorName string
+	KafkaTopic    string
+	Producer      *kafka.Producer
+}
+
+func getCEType(logReq LogRequest) (string, error) {
+	switch logReq.ReqType {
+	case InferenceRequest:
+		return CEInferenceRequest, nil
+	case InferenceResponse:
+		return CEInferenceResponse, nil
+	case InferenceFeedback:
+		return CEFeedback, nil
+	default:
+		return "", fmt.Errorf("Incorrect log request type: %s", errors.New("Incorrect log request type"))
+	}
+}
+
+func (w *Worker) sendKafkaEvent(logReq LogRequest) error {
+
+	reqType, err := getCEType(logReq)
+	if err != nil {
+		return err
+	}
+
+	kafkaHeaders := []kafka.Header{
+		{Key: KafkaTypeHeader, Value: []byte(reqType)},
```

Comment:
Does this mean that the headers will not be prefixed with `Ce-`? It would be good to standardise this, as currently MLServer does provide all kafka headers with the same prefx as cloud-events (ie with `Ce-` prefix

Reply:
The current open PR in MLServer that is